### DISCUSSION TITLE
Please Fix Pomegrenades

### DISCUSSION THREAD

**Discussion by Githubberclown** [FakeFishGames/Barotrauma#6910] [Ideas and suggestions]
You are hard limited to 3-5 pomegrenade plants because the moment the mutated ones spawn and explode they damage the pomegrenades by them making them effectively worthless

I have 2 ways to fix this That are extremely simple but this is super annoying and should have been known a while ago.

Make pomegrenades 100% immune to Explosive damage

OR

Make pomegrenades combinable past 100% locking them at full durability. (Example: You give a 50% pomegrenade to a 63% pomegrenade and it makes it 100% instead of 113%

**Comment by ghost** [comment]
I agree with maybe 90% of the second idea, making pomegrenades combineable could reduce the profits from pomegrenades without completely butchering them by making the fruit unuseable, though locking them at max durability would make them immune to fires and ruin the idea of keeping the collected fruit safely away from flames.
